stdlib/public/Glibc/Glibc.swift

```diff
@@ -74,3 +152,46 @@ public var SIG_HOLD: __sighandler_t {
 }
 #endif
 
+//===----------------------------------------------------------------------===//
+// semaphore.h
+//===----------------------------------------------------------------------===//
+
+/// The value returned by `sem_open()` in the case of failure.
+public var SEM_FAILED: UnsafeMutablePointer<sem_t> {
+  // The value is ABI.  Value verified to be correct on Linux.
```

Comment:
It would be better to say 'on Glibc'.  Other libc implementations on Linux can choose a different one, as far as I understand.